README.md

```diff
@@ -34,4 +35,4 @@ Vaadin recommends using the official Jandex index for the Pro components which i
     <artifactId>vaadin-jandex</artifactId>
 </dependency>
```

Comment:
This is a the maven dependency. it should be `implementation("com.vaadin:vaadin-jandex")` instead

Reply:
done